backend/lang/en/auth.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Authentication Language Lines
+    |--------------------------------------------------------------------------
+    |
+    | The following language lines are used during authentication for various
+    | messages that we need to display to the user. You are free to modify
+    | these language lines according to your application's requirements.
+    |
+    */
+
+    'failed' => 'These credentials do not match our records.',
+    'password' => 'The provided password is incorrect.',
+    'throttle' => 'Too many login attempts. Please try again in :seconds seconds.',
+    'logged_in' => 'Something wrong wrong',
```

Comment:
@ndvkhangsgt 
Please give the key name appropriate to the message.
Example:
```suggestion
    'login_fail' => 'Something wrong wrong',
```